web/src/util.js

```diff
@@ -45,10 +45,16 @@ export const validateEmail = (email) => {
     // eslint-disable-next-line
     var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
     return re.test(String(email).toLowerCase());
-};
+}
 
 export const validatePhoneNumber = (phoneNumber) => {
     // eslint-disable-next-line
     var re = /\(\d{3}\) \d{3}-\d{4}/;
     return re.test(phoneNumber);
+}
+
+export const getUserRole = () => {
```

Comment:
Sorry, I should have caught this earlier and commented.  Take a look at `credentialStore.js`, you'll want to use `getCredentials()` instead of this.  I realize I told you to use `util.js` so my fault.

The reasoning is that credentials may be in either local or session storage depending on whether the 'remember me' box was ticked on login.